app/controllers/sources.js

```diff
@@ -28,8 +73,8 @@ export default Ember.Controller.extend({
             type: 'GET',
             contentType: 'application/json',
         }).then((json) => {
-            this.set('numberOfSources', json.meta.pagination.count);
-            this.get('sources').addObjects(json.data);
+            let tmpSources = json.data.filter(source => sourcesWithData.includes(source.attributes.longTitle));
```

Comment:
This would be nicer if `sourcesWithData` were an object with titles as property names, so you could treat it like a set (`longTitle in sourcesWithData`) instead of iterating through the array over and over.